try {
null
}
catch(error) { console.log(error); }

